src/api/api.js

```diff
@@ -50,6 +50,11 @@ const usersAPI = {
       },
     });
   },
+
+  saveProfile: (profile) => {
```

Comment:
Don't forget about arrow functions short form:
`() => { return ... }` might be written shorter `() => ...`
Arrow functions in the short form without {} will immediately return what's written after =>.
`saveProfile: (profile) => instance.put(`${ENDPOINTS.SAVE_PROFILE}`, profile),` looks much better.

Reply:
done